web_ui/authentication.go

```diff
@@ -274,6 +324,27 @@ func resetLoginHandler(ctx *gin.Context) {
 	}
 }
 
+func whoamiHandler(ctx *gin.Context) {
```

Comment:
Maybe add comments before this function as well, just to remain consistent even though it is pretty self-explanitory